internal/rbd/manager.go

```diff
@@ -237,6 +237,21 @@ func (mgr *rbdManager) GetVolumeGroupByID(ctx context.Context, id string) (types
 	return vg, nil
 }
 
+func (mgr *rbdManager) MakeVolumeGroupID(ctx context.Context, poolID int64, name string) (string, error) {
+	clusterID, err := util.GetClusterID(mgr.parameters)
+	if err != nil {
+		return "", fmt.Errorf("failed to get cluster-id: %w", err)
+	}
+
+	// convert the poolid + name to an id/handle
```

Comment:
`// convert the clusterid + poolid + name to an id/handle`

Reply:
done.